plugins/coverbrowser.koplugin/listmenu.lua

```diff
@@ -502,6 +493,9 @@ function ListMenuItem:update()
                         authors = { authors[1], T(_("%1 et al."), authors[2]) }
                     end
                     authors = table.concat(authors, "\n")
+                    -- Needs to fit 3 lines instead of 2
+                    fontsize_title = _fontSize(17)
+                    fontsize_authors = _fontSize(15)
```

Comment:
3 lines of authors or 3 lines in total (1 line of title + 2 line of authors) ?
Because we already could display 2 line of authors, and I don't really see the need to display 3:

<kbd>![image](https://user-images.githubusercontent.com/24273478/67003605-a1d4e200-f0de-11e9-9f53-ccd26c53386e.png)</kbd>

Or is it that you prefer to decrease the font size in that case to have more blank space around? (I felt you initially disliked that :)
Dunno about how you prefer it with series, but in my screenshot case, I think I prefer it as on the screenshot.

edit: tested, and I don't see anything changing because of that: I guess we'd already go to some smaller font size than the one you set here when having to display 1+2 lines.
So why did you have to do that? Some particular case I didn't manage to test? Or is it just to start decreasing font size from a lower start value to avoid some loops?

Reply:
When displaying title and 2 authors, or title, series, and author, the loop would run 3 times, ending with title font size 17 and author font size 15. So I decided to not cause the 3 loops and go to the right font size immediately.
Thanks to your input to this PR, we can always compute spacing on a 64px height, and all scaling to the right sizes will be applied afterwards.

Reply:
You might want to make that comment `-- Needs to fit 3 lines instead of 2` more explicite then, like `-- as we'll fit 3 lines instead of 2, we can avoid some loops by starting from a lower font size`